src/main/java/games/strategy/engine/framework/startup/ui/MetaSetupPanel.java

```diff
@@ -9,14 +9,24 @@
 import javax.swing.JOptionPane;
 import javax.swing.JPanel;
 
+import games.strategy.engine.config.client.LobbyServerPropertiesFetcher;
 import games.strategy.engine.framework.GameRunner;
+import games.strategy.engine.framework.ProcessRunnerUtil;
 import games.strategy.engine.framework.startup.mc.SetupPanelModel;
 import games.strategy.engine.lobby.client.LobbyClient;
 import games.strategy.engine.lobby.client.login.LobbyLogin;
+import games.strategy.engine.lobby.client.login.LobbyServerProperties;
 import games.strategy.engine.lobby.client.ui.LobbyFrame;
 import games.strategy.triplea.UrlConstants;
+import games.strategy.triplea.settings.ClientSetting;
 import games.strategy.ui.SwingComponents;
+import swinglib.JButtonBuilder;
+import tools.map.making.MapCreator;
 
+/**
+ * This is the main welcome panel with 'play online' buttons etc...
+ * The panel does not include the 'play
+ */
```

Comment:
Looks like you may have been interrupted while editing this comment (see 1e8e282). :smile: